### PR TITLE
Add bug bash docs

### PR DESCRIPTION
**Pull request by TerryHowe**

### CODE REVIEW

File: docs/bug_bash.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+
+
+EKS Anywhere curated packages is a framework to manage installation, configuration and maintenance of components that provide general operational capabilities for Kubernetes applications. This guide describe the steps to install EKS Anywhere package controller and install, configure, upgrade and delete a package.
+
+## Before you start
+
+You must create a EKS Anywhere cluster The easiest thing to do is create a (local EKS Anywhere cluster)[https://anywhere.eks.amazonaws.com/docs/getting-started/local-environment/] on your Mac or Linux machine using the Docker provider.
```
Comment (ewollesen) [inline docs/bug_bash.md:7]:
```suggestion
You must create a EKS Anywhere cluster. The easiest thing to do is create a [local EKS Anywhere cluster](https://anywhere.eks.amazonaws.com/docs/getting-started/local-environment/) on your Mac or Linux machine using the Docker provider.
```

File: docs/bug_bash.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+        kubectl port-forward harbor-something
+
+Once you have create the port forward, connect to it using the domain you create (https://my-harbor.local)[https://my-harbor.local]. Verify the version of Harbor installed is what you expected.
+
+## Upgrade Harbor
+
+In order to upgrade harbor, run the following command:
+
+        kubectl upgrade package my-harbor v2.4.2-c47074c1d3d92a2359f6b379e6688c323338ec18-helm
```
Comment (ewollesen) [inline docs/bug_bash.md:69]:
```suggestion
        eksctl anywhere upgrade package my-harbor v2.4.2-c47074c1d3d92a2359f6b379e6688c323338ec18-helm
```

File: docs/bug_bash.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+        chmod 0755 eksctl-anywhere
+        mv eksctl-anywhere $(type -f eksctl-anywhere| cut -d' ' -f3)
+
+## Listing the available packages
+
+You can get a list of the packages that are available for install using
+
+        eksctl anywhere list packages
+
+## Install a package
```
Comment (a-cool-train) [inline docs/bug_bash.md:29]: Is this for installing packages after cluster creation?

File: docs/bug_bash.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+## Before you start
+
+You must create a EKS Anywhere cluster The easiest thing to do is create a [local EKS Anywhere cluster](https://anywhere.eks.amazonaws.com/docs/getting-started/local-environment/) on your Mac or Linux machine using the Docker provider.
+
+## Install package controller
+
+Right now, EKS Anywhere Packages is installed using a helm chart. 
+
+         helm install cert-manager oci://public.ecr.aws/j0a1m4z9/aws/eks-anywhere-packages --version v0.1.0
```
Comment (binyamz) [inline docs/bug_bash.md:13]: cert-manager as a name will be confusing. let's name it "packages" or "package-manager"

File: docs/bug_bash.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+        apiVersion: packages.eks.amazonaws.com/v1alpha1
+        kind: Package
+        metadata:
+          name: my-harbor
+        spec:
+          packageName: Harbor
+          packageVersion: v2.4.1-c47074c1d3d92a2359f6b379e6688c323338ec18-helm
+          config:
+            certificate: ASDF
```
Comment (binyamz) [inline docs/bug_bash.md:47]: Documentation should include on how to generate certificates or any required perquisites for harbor.

File: docs/bug_bash.md (hunk 3 above)
Comment (TerryHowe) [reply docs/bug_bash.md:29]: Yes. If the CLI is ready for cluster creation, we could have instructions for both scenarios

File: docs/bug_bash.md (hunk 4 above)
Comment (TerryHowe) [reply docs/bug_bash.md]:
```suggestion
         helm install package-manager oci://public.ecr.aws/j0a1m4z9/aws/eks-anywhere-packages --version v0.1.0
```

Copy pasta mistake

**Review by jonahjon — APPROVED** [review]

**Comment by jonahjon** [comment]
I'm ok with merging a starting place for docs.

**Pull request closed without merging**